app/models/rubygem.rb

```diff
@@ -321,7 +321,7 @@ def protected_gem_typo
     gem_typo = GemTypo.new(name)
 
     return unless gem_typo.protected_typo?
-    errors.add :name, "'#{name}' is too close to typo-protected gem: #{gem_typo.protected_gem}"
+    errors.add :name, "Your gem '#{name}' is too similar to typo-protected gem named '#{gem_typo.protected_gem}'"
```

Comment:
Please change `type-protected gem` to `an existing gem`. There is no non-typo-protected gem.

Reply:
Rewrote error message on branch, thank you for the suggestion!